.circleci/config.yml

```
@@ -166,7 +166,7 @@ jobs:
             - packages/react-instantsearch-nextjs/dist
             - packages/vue-instantsearch/vue2
             - packages/vue-instantsearch/vue3
-
+            - packages/instantsearch-ui-components/dist
```

```suggestion
            - packages/instantsearch-ui-components/dist

```

nbd but there was a separator here before